subsys/bluetooth/mesh/Kconfig

```diff
@@ -129,16 +256,16 @@ config BT_MESH_PROVISIONER
 	help
 	  Enable this option to have support for provisioning remote devices.
 
-config BT_MESH_CDB
-	bool "Mesh Configuration Database"
-
 config BT_MESH_ECDH_P256_CMAC_AES128_AES_CCM
```

Comment:
Shouldn't this option be wrapped up in mesh-1.1 condition?

Reply:
Added `if BT_MESH_V1d1` to the prompt. Please check if it is correct.

Reply:
I'd recommend moving this option to this: `config BT_MESH_ECDH_P256_HMAC_SHA256_AES_CCM`.
Seems they do similar things but located in different places. 
Or group these three options somewhere together:
```
config BT_MESH_ECDH_P256_CMAC_AES128_AES_CCM
config BT_MESH_ECDH_P256_HMAC_SHA256_AES_CCM
config BT_MESH_OOB_AUTH_REQUIRED
```

Reply:
I mean move it under ...1d1 option or move to provisioning menu with conditional prompt. This is up to you

Reply:
Moved, but kept `bool ... if BT_MESH_V1d1` otherwise this option disappears from the final `.config` file and 1.0 device doesn't get the only supported algorithm enabled.